lib/presentation/pages/widgets/game_form.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:guess_the_number/constants/global.dart';
+import 'package:guess_the_number/presentation/bloc/events.dart';
+import 'package:guess_the_number/presentation/bloc/game_bloc.dart';
+import 'package:guess_the_number/presentation/bloc/state.dart';
+
+class GameForm extends StatelessWidget {
+  final GameState state;
+  static final _formKey = GlobalKey<FormState>();
+
+  const GameForm({Key? key, required this.state}) : super(key: key);
+
```

Comment:
I would put (_validator) in a block and get it through the provider.

Reply:
may be.. let @sergei-mikhailovskii-idf  to decide